cardigan_beta/stories/content.js

```diff
@@ -4,9 +4,13 @@ function randomNumber(min, max) {
   return Math.floor(Math.random() * max) + min;
 }
 
-export const image = () => {
-  const contentUrl = faker.image.image();
+export const id = randomNumber(1000, 2000);
+
+export const url = faker.internet.url();
 
+export const image = () => {
+  // const contentUrl = faker.image.imageUrl(800, 450, 'city'); // lorempixel taking too long to load
```

Comment:
?

Reply:
faker users lorempixel for it's image urls, but it's really slow, so better not to use. I've removed these comments in #2780